### PR TITLE
feat(config): add Zooz ZSE43 and ZSE44 device config files

### LINKED ISSUE #3971: Missing device configuration: Zooz ZSE43
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Zooz ZSE43

### Manufacturer ID

0x027a

### Product Type

0x7000

### Product ID

0x7000

### Firmware Version

1.10

### Is the device listed on the Z-Wave Alliance website?

Not yet

### Do you have a manual?

[zooz-700-series-tilt-shock-xs-sensor-zse43-manual.pdf](https://github.com/zwave-js/node-zwave-js/files/7781816/zooz-700-series-tilt-shock-xs-sensor-zse43-manual.pdf)

### PR DESCRIPTION
**Pull request by Botched1**
Add config file for Zooz zse43 tilt/shock sensor

Closes #3971

### CODE REVIEW

**Comment by Botched1** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x027a/zse43.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Shock On/Off",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zse43.json:33]: In the production firmware, this isn't an enable/disable parameter and instead has three options. Please update.

https://www.support.getzooz.com/kb/article/817-zse43-tilt-shock-xs-sensor-advanced-settings/

File: packages/config/config/devices/0x027a/zse43.json [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+				{
+					"label": "Low",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Group 2: On Delay",
+			"valueSize": 2,
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zse43.json]: This should be size 4.

File: packages/config/config/devices/0x027a/zse43.json [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 3600,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "Group 2: Off Delay",
+			"valueSize": 2,
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zse43.json]: This should be size 4 as well.

File: packages/config/config/devices/0x027a/zse44.json [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "14",
+			"label": "Temperature Offset - 0=-10, 100=0, 200=+10",
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zse44.json]: This is a particularly screwy parameter, but put that information in the description not the label.

File: packages/config/config/devices/0x027a/zse44.json [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+			"valueSize": 1,
+			"unit": "0.1 Degrees",
+			"minValue": 0,
+			"maxValue": 200,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+		{
+			"#": "15",
+			"label": "Humidity Offset - 0=-10, 100=0, 200=+10",
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zse44.json]: Same, the description is the right place for further explanation.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Great job on this so far. A few minor comments and one parameter that I suspect changed in the production firmware based on what Zooz sent me.

**Review by blhoward2 — DISMISSED** [review]

**Review by AlCalzone — APPROVED** [review]
OK with my changes

**Pull request merged**